armi/reactor/blueprints/__init__.py

```diff
@@ -402,7 +402,7 @@ def _resolveNuclides(self, cs):
                 newNuclides = expansions[elemental]
                 # overlay code details onto nuclideFlags for other parts of the code
                 # that will use them.
-                # CRAP: would be better if nuclideFlags did this upon reading s.t.
+                # TODO: would be better if nuclideFlags did this upon reading s.t.
```

Comment:
:D

Reply:
heh. Admittedly, that's not an important change, and I will happily revert it.